assets/chat/css/onstream.scss

```diff
@@ -67,6 +78,7 @@ i[class^="icon-"],
 a.user {
     background-color: transparent;
     background-position-x: -200px;
+    background-image: none !important;
```

Comment:
you can't override the background-image because it's set on the element... setting the background position should hide the viewer state indicator, what was this meant to fix?

Reply:
adding `!important` overrides an element's style

Reply:
also changing the background position can lead to weird bugs with longer names

Reply:
!important does not override inline styles

Reply:
if the background position wasn't offset by enough to handle long names (200px should be more than enough but whatever...) you could increase the offset